src/nss-systemd/nss-systemd.c

```diff
@@ -250,7 +250,7 @@ static enum nss_status copy_synthesized_group(
         /* String fields point into the user-provided buffer */
         dest->gr_name = buffer;
         dest->gr_passwd = stpcpy(dest->gr_name, src->gr_name) + 1;
-        dest->gr_mem = (char **) strcpy(dest->gr_passwd, src->gr_passwd) + 1;
+        dest->gr_mem = (char **) stpcpy(dest->gr_passwd, src->gr_passwd) + 1;
```

Comment:
uh, how did this ever work?

Reply:
I guess this did not work. We do not have enough tests for nss modules...